interfaces/database/user_repository.go

```diff
@@ -23,16 +36,41 @@ func (repo *UserRepository) FindById(identifier int) (domain.User, error) {
 	row, err := repo.Query("SELECT * FROM users WHERE id = ?", identifier)
 	user := domain.User{}
 	defer row.Close()
+
 	if err != nil {
+		log.Fatal(err)
 		return user, err
 	}
+
 	var id int
 	var amount int
 	row.Next()
-	if err = row.Scan(&id, &amount); err != nil {
+	if err = row.Scan(
+		&id,
+		&amount,
+		TrashScanner{},
+	); err != nil {
+		log.Fatal(err)
 		return user, err
 	}
+
 	user.ID = id
 	user.Amount = amount
 	return user, nil
-}
\ No newline at end of file
+}
+
+//Points :adds points to User.
+func (repo *UserRepository) Points(u domain.User) (id int, err error) {
+	user, err := repo.FindById(u.ID)
+	var newAmount int
+	newAmount = int(user.Amount) + u.Amount
+
+	_, err = repo.Execute("UPDATE users SET amount = ? WHERE id = ?", newAmount, user.ID)
```

Comment:
```suggestion
	_, err = repo.Execute("UPDATE users SET amount =  amount + ? WHERE id = ?", u.Amount, u.ID)
```

このクエリにすることで、↑のFindByIdのクエリを不要にすることが出来るかと思います。